tests/testthat/_snaps/type.md

```diff
@@ -125,7 +125,7 @@
       vec_ptype_common(foo = TRUE, !!!list(bar = 1, "foo"))
     Condition
       Error:
-      ! Can't combine `bar` <double> and `..3` <character>.
+      ! Can't combine `foo` <double> and `..3` <character>.
```

Comment:
This is mildly confusing

Reply:
Agreed, this is a separate issue though. In some cases we could just refer to the latest input. It can get complicated because input 1 and 2 might create a new type, and then there is nothing to refer to if that new type is not compatible with input 3.